parmed/gromacs/gromacstop.py

```diff
@@ -491,10 +491,10 @@ def _parse_atoms(self, line, params):
             atom = Atom(atomic_number=atomic_number, name=words[4],
                         type=words[1], charge=charge, mass=mass)
         # check for insertion code
-        if words[2].isnumeric():
+        try:
```

Comment:
It looks like the problem is that a leading `-` leads to `isnumeric()` returning `False`.  What about we change line 494 to this instead:

```python
if words[2].isnumeric() or (words[2].startswith('-') and words[2][1:].isnumeric()):
```

Reply:
Done via [a8d0e3f](https://github.com/ParmEd/ParmEd/pull/1222/commits/a8d0e3fed7279bbcaba5f7453a70363325ff24c1)